### ISSUE TITLE
OutOfRange Error

### ISSUE THREAD

**Issue by JproChen** [tynguyen/unsupervisedDeepHomographyRAL2018#22]
OutOfRangeError (see above for traceback): RandomShuffleQueue '_1_shuffle_batch/random_shuffle_queue' is closed and has insufficient elements (requested 128, current size 0)
         [[node shuffle_batch (defined at D:\projects\unsupervisedDeepHomographyRAL2018\code\dataloader.py:233)  = QueueDequeueManyV2[component_types=[DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_INT32], timeout_ms=-1, _device="/job:localhost/replica:0/task:0/device:CPU:0"](shuffle_batch/random_shuffle_queue, shuffle_batch/n/_141)]]

Is there something wrong with the data loader? or should I generate specific number of training data?

**Comment by mcmate** [comment]
same error, thank you in advance

**Comment by tynguyen** [comment]
I'm not so sure what happened. Some of these might help? 
https://stackoverflow.com/questions/34050071/tensorflow-random-shuffle-queue-is-closed-and-has-insufficient-elements/43370673
